src/Raven.Server/ServerWide/SecretProtection.cs

```diff
@@ -325,9 +328,117 @@ private static byte[] DecryptProtectedData(byte[] secret, byte[] key)
             return unprotectedData;
         }
 
+        private static void ValidateExpiration(string source, X509Certificate2 loadedCertificate, LicenseType licenseType, SetupProgressAndResult progress = null)
+        {
+            if (loadedCertificate.NotAfter < DateTime.UtcNow)
+            {
+                string msg = $"The provided certificate '{loadedCertificate.FriendlyName}' from {source} is expired! Thumbprint: {loadedCertificate.Thumbprint}, Expired on: {loadedCertificate.NotAfter}";
+                if (Logger.IsOperationsEnabled)
+                    Logger.Operations(msg);
+                progress?.AddError(msg);
```

Comment:
We log but we don't throw? The original code also had this issue
![image](https://user-images.githubusercontent.com/1541805/160126477-2808b846-221c-444f-a308-0a2f7150137e.png)

Reply:
thanks
fixed